parsers/uottawa.js

```diff
@@ -7,7 +7,7 @@ var fs = require('fs')
   });
 
 // Relevant regular expressions
-var _locationRegex = /\b[A-Z]{3,5}\s[A-Z0-9]{2,4}\b/;
+var _locationRegex = /\b[A-Z]{3,5}\s[A-Z0-9]{2,6}\b/;
```

Comment:
`var _courseRegex = /\b([A-Z]{3})([1-9]{1}[0-9]{3})([A-Z])?\s+(\b([A-Z.]+\s([A-Z.&]+\s?)+)\b)/;`

first set of () is for the course code, second is for the course name, which is an arbitrary number of capital letters followed by a space and more capital letters, so on and so forth. "SEE CLASS NOTES" would be supported.